test/unit_tests/controllers/account_controller_test.dart

```diff
@@ -47,7 +58,7 @@ void main() {
       await accountController.getLinkedAccounts();
 
       // Check if the repository function was called with right arguments
-      verify(accountRepository.getUserAccounts(authController.user.id));
+      verify(accountRepository.getUserAccounts(any));
```

Comment:
little confused why you're calling it with any

Reply:
not sure either, I'll revert that